fundamentals/minimal-apis/samples/IResultImplementation/Tests/IResultImplementationTest.cs

```diff
@@ -88,6 +89,11 @@ public void CreateTodoSavesContactToDatabase()
             var mockDbContextOptions = new Mock<DbContextOptions<IResultImplementationContext>>();
             var mockContext = new Mock<IResultImplementationContext>();
             mockContext.Setup(c => c.Contact).Returns(MockSet!.Object);
+            if(Data == null)
+            {
+                Assert.Fail("Data null");
+                return;
+            }
```

Comment:
I don't think this is necessary since it is a controlled environment.  A null-forgiving operator would be better I guess 😅

Reply:
Why kill a mosquito with a swatter when you can use a bazooka? 

OK, review now.

Reply:
Nice line though 😅